server/routes/createGoal/routerRequestHandlers.test.ts

```diff
@@ -211,4 +212,24 @@ describe('routerRequestHandlers', () => {
       expect(next).not.toHaveBeenCalled()
     })
   })
+
+  describe('checkAddNoteFormExistsInSession', () => {
+    it(`should redirect to Create Goal screen given add note form does not exist in session`, async () => {
```

Comment:
`should redirect to Add Note screen given add note form does not exist in session` ?